packages/sanity/src/_internal/cli/actions/backup/downloadAsset.ts

```diff
@@ -0,0 +1,52 @@
+import path from 'path'
+import {createWriteStream} from 'fs'
+import withRetry from './withRetry'
+import debug from './debug'
+
+const {getIt} = require('get-it')
+const {keepAlive, promise} = require('get-it/middleware')
+
+const CONNECTION_TIMEOUT = 15 * 1000 // 15 seconds
+const READ_TIMEOUT = 3 * 60 * 1000 // 3 minutes
+
+const request = getIt([keepAlive(), promise()])
+
+async function downloadAsset(
+  url: string,
+  fileName: string,
+  fileType: string,
+  outDir: string,
+): Promise<void> {
+  // File names that contain a path to file (sanity-storage/assets/file-name) fail when archive is created, so we
+  // want to handle them by taking the base name as file name.
+  const normalizedFileName = path.basename(fileName)
```

Comment:
I don't quite follow this part, can you give me an example?

Reply:
Updated inline comment:
```
  // File names that contain a path to file (e.g. sanity-storage/assets/file-name.tar.gz) fail when archive is
  // created due to missing parent dir (e.g. sanity-storage/assets), so we want to handle them by taking
  // the base name as file name.
```
Does this make sense?

Reply:
Oh, I understand now. I think this is from when the API was returning a path early on but now it only returns a filename. So I think you could remove this if you like. Up to you.

Reply:
I vote to keeping it even if server does not return paths in file name now. Without this CLI fails hard on a slash.